config/stage.json

```diff
@@ -59,7 +59,7 @@
 		"jetpack/pricing-page-annual-only": true,
 		"jetpack/simplify-pricing-structure": false,
 		"jetpack/standalone-plugin-onboarding-update-v1": true,
-		"jetpack-social/advanced-plan": false,
+		"jetpack-social/advanced-plan": true,
```

Comment:
I noticed that the URL after clicking on my plans from Jetpack settings is the calypso blue URL and not the calypso URL. Therefore, enabling this feature flag for proxied version of calypso blue, just like how we did it for calypso green.